lib/ex_unit_fixtures.ex

```diff
@@ -16,22 +16,24 @@ defmodule ExUnitFixtures do
   Next you should:
 
   1. Add `use ExUnitFixtures` to your test cases (before `use ExUnit.Case`)
-  2. Define some fixtures using `deffixture/3`
-  3. Tag some tests with `@tag fixtures: [:your_fixtures_here]`
+  2. Add `ExUnit.Case.register_attribute __MODULE__, :fixtures` (after `use ExUnit.Case`)
```

Comment:
It''s a shame to introduce this as a requirement.  Been trying to think up a way to avoid it, but not got anything yet...